conbench/entities/benchmark_result.py

```diff
@@ -352,6 +361,86 @@ def to_dict_for_json_api(benchmark_result):
             },
         }
 
+    def is_failed(self):
+        """
+        Return True if this BenchmarkResult is considered to be 'failed' /
+        erroneous.
+
+        The criteria are conventions that we (hopefully) apply consistently
+        across components.
+        """
+        if self.data is None:
+            log.info("data is none")
+            return True
+
+        if do_iteration_samples_look_like_error(self.data):
+            log.info("iterations look like err")
+            return True
+
+        if self.error is not None:
+            log.info("selferr is not none")
+            return True
+
+        return False
+
+    @property
+    def svs(self) -> float:
+        """
+        Return single value summary (currently: mean) or raise an
+        Exception.
+        """
+
+        # This is here just for the shorter name.
+        return self._single_value_summary()
+
+    @property
+    def svs_type(self) -> str:
+        """
+        Return single value summary type (currently: "mean")
+        """
+        return "mean"
```

Comment:
I like this idea—it makes it easy for us to switch to median or min/max or whatever later if we want. The name is a bit cryptic, though it gets used enough that it is nice if it's short.

Reply:
Thank you for expressing positivity here. Often, it's really just about making an interface nicer (while not changing user-facing functionality). This was/is my attempt to do so, start building an interface that makes it a little easier for us to reason about. I am glad that you think this goes into a good direction.

About naming: yes: I thought about terminology quite a bit, and I think the 'svs' shortcut might be a nicely recognizable thing to have. 

> if it gets used enough that it is nice if it's short.

Cool. Yeah, let's try it out!